src/containers/org/members/PeopleTable.js

```diff
@@ -0,0 +1,294 @@
+    250
+  ), []);
+
+  const handleOnChangeMemberFilterQuery = (event :SyntheticInputEvent<HTMLInputElement>) => {
+    debouncedFilterQuery(event.target.value);
+  };
+
+  const handleUnassignRole = (member :Map, role :Role) => {
+    setTargetMember(member);
+    setTargetRole(role);
+    setIsVisibleRemoveRoleFromMemberModal(true);
+  };
+
+  const handleUnassignAllRoles = () => {
+    // TODO: unassign all roles from target member
+  };
+
+  const handleRemoveMember = (member :Map) => {
+    setTargetMember(member);
+    setIsVisibleRemoveMemberFromOrgModal(true);
+  };
+
+  const handleAddMember = () => {
+    setIsVisibleAddMemberToOrgModal(true);
+  };
+
+  const handleOnFilterChange = (category :string, id :string) => {
+    const categoryFilters = selectedFilters.get(category, Set());
+    const newFilters = categoryFilters.includes(id)
+      ? categoryFilters.remove(id)
+      : categoryFilters.add(id);
+
+    setSelectedFilters(selectedFilters.set(category, newFilters));
+  };
+
+  const handleSelectMember = (member :Map) => {
+    const { id } = getUserProfile(member);
+    const newSelection = selectedMembers.has(id)
+      ? selectedMembers.remove(id)
+      : selectedMembers.set(id, member);
+    setSelectedMembers(newSelection);
+  };
+
+  const onSelectAll = () => {
+    const newSelection = Map().withMutations((mutable) => {
+      filteredMembers.forEach((member) => {
+        const { id } = getUserProfile(member);
+        mutable.set(id, member);
+      });
+    });
+
+    setSelectedMembers(newSelection);
+  };
+
+  const onDeselectAll = () => {
+    setSelectedMembers(Map());
+  };
+
+  const selectionText = selectedMembers.size
+    ? `${selectedMembers.size} selected`
+    : `${filteredMembers.size} members`;
+
+  return (
+    <div>
+      <TableToolbar>
+        <MembersCheckboxWrapper>
+          {/* TODO: Support indeterminate checkbox state */}
+          <Checkbox
+              checked={selectedMembers.size}
+              onChange={selectedMembers.size ? onDeselectAll : onSelectAll} />
+          <Selection>{selectionText}</Selection>
+        </MembersCheckboxWrapper>
+        <BulkActionButton onAddRolesClick={() => setIsVisibleAssignRolesModal(true)} />
+        <SearchInput onChange={handleOnChangeMemberFilterQuery} />
+        <FilterButton
+            filter={selectedFilters}
+            onFilterChange={handleOnFilterChange}
+            organizationId={organizationId}
+            roles={roles} />
+        <Button
+            color="primary"
+            onClick={handleAddMember}
+            startIcon={PlusIcon}>
+          Add Member
+        </Button>
+      </TableToolbar>
+      <FilterChipsList
+          filters={selectedFilters}
+          onDelete={handleOnFilterChange}
+          roles={roles} />
+      <Table cellPadding="0" cellSpacing="0">
+        <colgroup>
+          <col width="56px" />
+          <col width="268px" />
+          <col width="150px" />
+        </colgroup>
```

Comment:
What's this?

Reply:
This assigns the width of the columns at the table level without having each row decide for itself.